React/Views/RCTMaskedView.m

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
+#import "RCTMaskedView.h"
+#import "UIView+React.h"
+
+@implementation RCTMaskedView
+
+- (void)didUpdateReactSubviews
+{
+  [super didUpdateReactSubviews];
```

Comment:
Yeah, if you want clipped subviews you can always wrap the children of your MaskedView yourself.